lang/de_DE.po

```diff
@@ -4899,7 +4899,7 @@ msgstr "%s ist nur mit dem Protokoll TCP gültig"
 
 #, php-format
 msgid "%s is only valid if the gateway is set to 'default'."
-msgstr "%s ist nur gültig, wenn Gateway auf 'default' gesetzt wird."
+msgstr "%s ist nur gültig, wenn Gateway auf 'Standard' gesetzt wird."
```

Comment:
gibt es - dieser string wird auf der Seite zum bearbeiten von Firewallregeln verwendet - wird in der auswahlliste aber klein geschrieben - werde es hier anpassen